di-ipv-evcs-stub/lambdas/src/domain/enums/vcState.ts

```diff
@@ -0,0 +1,26 @@
+enum VcState {
+  CURRENT = "CURRENT",
+  PENDING = "PENDING",
+  VERIFICATION = "VERIFICATION",
+  ABANDONED = "ABANDONED",
+  PENDING_RETURN = "PENDING_RETURN",
+  HISTORIC = "HISTORIC",
+  VERIFICATION_ARCHIVED = "VERIFICATION_ARCHIVED",
+}
+
+export const CreateVcStates = {
+  [VcState.CURRENT]: VcState.CURRENT,
+  [VcState.PENDING]: VcState.PENDING,
+  [VcState.PENDING_RETURN]: VcState.PENDING_RETURN,
+  [VcState.VERIFICATION]: VcState.VERIFICATION,
+} as const;
```

Comment:
Can this not be something like:
```suggestion
export const isValidCreateVcState = (stateToCheck) => stateToCheck in VcState;
```
Or do we need to construct an object in order to do this? (genuine question, I could be mistaken)

Reply:
I believe when you compile the enum in typescript it already creates a JS object representation of the enum that looks like

Reply:
Can't use export const isValidCreateVcState = (stateToCheck) => stateToCheck in VcState; 
as the const declared are like sub VcState enum .....contain VcState's valid for create or update.